db/migrate/20250731154006_add_notify_parents_to_vaccination_records.rb

```diff
@@ -0,0 +1,18 @@
+# frozen_string_literal: true
+
+class AddNotifyParentsToVaccinationRecords < ActiveRecord::Migration[8.0]
+  def up
+    add_column :vaccination_records, :notify_parents, :boolean
```

Comment:
Should we default this to `false`? I'm wondering if there's any benefit to it being able to be both `nil` and `false`?

Reply:
From the domain point of view, a default of `true` is more appropriate (as **not** notifying the parents is an exception).

Reply:
I was wondering about that too. I suggested `false` because sending a confirmation to parents who's patient explicitly asked them not to receive is perhaps worse than accidentally not sending a confirmation in normal circumstances?